dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/internal/LibertyGradle.java

```diff
@@ -76,57 +73,79 @@ public static LibertyBuildPluginConfiguration getLibertyGradleProjectConfigurati
         return instance.getProjectInspector(project, monitor).getBuildPluginConfiguration(monitor);
     }
 
-    // TODO - Issue 52
-    public static boolean runGradleCommand(final IPath workingDir, String options, List<String> profiles, IProgressMonitor monitor) {
+    /**
+     * Run a set of Gradle tasks
+     * @param workingDir The base Gradle project directory
+     * @param tasks The tasks to run
+     * @param args The arguments for the tasks
+     * @param monitor A progress monitor that can be used to cancel the tasks
+     * @return True if the tasks ran successfully, false otherwise
+     */
+    public static boolean runGradleTask(final IPath workingDir, String[] tasks, String[] args, IProgressMonitor monitor) {
```

Comment:
Should we pass a List instead of an array?

Reply:
I used an array to force the code to create a list or make a new array in order to add to it (since something like LibertyGradleConstants.SKIP_TESTS_ARGS should not be modified. Also the Gradle tooling API takes arrays.